test/executor.test.js

```diff
@@ -52,6 +52,39 @@ describe('executor', function() {
     }
   });
 
+  describe('when booting', function() {
+    it('should set the booting status', function(done) {
+      expect(app.booting).to.be.undefined;
```

Comment:
Please use `.to.be.undefined()` to prevent jshint warnings.

```
$ npm test

> loopback-boot@2.5.2 pretest /Users/bajtos/src/loopback/boot
> jscs . && jshint .

No code style errors found.
test/executor.test.js: line 57, col 33, Expected an assignment or function call and instead saw an expression.
test/executor.test.js: line 59, col 27, Expected an assignment or function call and instead saw an expression.
test/executor.test.js: line 60, col 35, Expected an assignment or function call and instead saw an expression.
test/executor.test.js: line 67, col 28, Expected an assignment or function call and instead saw an expression.
test/executor.test.js: line 71, col 27, Expected an assignment or function call and instead saw an expression.
test/executor.test.js: line 77, col 27, Expected an assignment or function call and instead saw an expression.

6 errors
npm ERR! Test failed.  See above for more details.
```